docs/migrating/migrating_from_ez_publish.md

```diff
@@ -44,6 +44,11 @@ If you are coming directly from legacy (4.x), you need to follow the best practi
 
 See Upgrade documentation on how to perform the actual upgrade: [Upgrade (eZ Publish Platform page)](https://doc.ez.no/display/EZP/Upgrade)
 
+!!! caution "Avoid exception when migrating the database"
+
+    If you plan to migrate from from eZ Publish through eZ Publish Platform 5.4 to eZ Platform and further, an exception may occur when you try to migrate the database while it contains internal drafts of Landing Pages, because such drafts do not have an expected row in the `ezcontentobject_name` table. 
+    To overcome this obstacle, you must first remove all internal drafts by setting the `InternalDraftsCleanUpLimit` and `InternalDraftsDuration` values to 0 and [running](https://doc.ez.no/eZ-Publish/Technical-manual/4.x/Features/Cronjobs/Running-cronjobs) the [internal drafts cleanup](https://github.com/ezsystems/ezpublish-legacy/blob/2019.03/cronjobs/internal_drafts_cleanup.php) cron job. 
```

Comment:
```suggestion
    To avoid this exception, you must first remove all internal drafts by setting the `InternalDraftsCleanUpLimit` and `InternalDraftsDuration` values to 0 and [running](https://doc.ez.no/eZ-Publish/Technical-manual/4.x/Features/Cronjobs/Running-cronjobs) the [internal drafts cleanup](https://github.com/ezsystems/ezpublish-legacy/blob/2019.03/cronjobs/internal_drafts_cleanup.php) cron job. 
```
The old 4.x doc is planned to be disabled soon-ish. Can we handle it somehow without linking to it?